app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -36,6 +37,12 @@ export const isOperandSearch = word => {
 
 const containsInArray = (word, array) => array.some(operand => word.includes(operand))
 
+export const canCloseColumn = projectBoard =>
+  values(projectBoard.visibleColumns).filter(Boolean).length > 1;
```

Comment:
You can make it even simpler with underscore's [`some`](https://underscorejs.org/#some):

```js
import { some } from 'underscore';

// ...

some(projectBoard.visibleColumns);
```

Reply:
I have tried with this method, but does not work how expected, because the `projectBoard.visibleColumns` is a object. So we need to do:
`some( values(projectBoard.visibleColumns))`, but does not work because in the moment of this function is called, the columns that are going to be closed is `true`, and we need to know if have other column (disregarding this column) are opened. And this is the reason of our conditional be `length > 1` instead of `length >= 1`